Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2601,6 +2630,10 @@ - (void)imagePickerController:(UIImagePickerController *)picker
             return failedToPickAttachment(nil);
         }
 
+        // Images chosen from the "attach document" UI should be sent as originals;
+        // images chosen from the "attach media" UI should be resized to "medium" size;
+        TSImageQuality imageQuality = (self.isPickingMediaAsDocument ? TSImageQualityOriginal : TSImageQualityMedium);
```

Comment:
I've made this change for images - I wonder if we should do something similar for videos?